integration-tests/test_collection.py

```diff
@@ -39,15 +83,45 @@ def test_output_file_non_existing_path(insights_client):
 
 
 def test_output_dir_without_specifying_a_path(insights_client):
-    """Test --output-dir (without specifying path)"""
+    """
+    :id: 63eb1ba0-b9dd-4185-b422-18325c12b503
+    :title: Test Output Directory Without Specifying a Path
+    :description:
+        Verifies that the --output-dir option fails with an appropriate
+        error message when no path is specified
+    :tier: Tier 1
+    :steps:
+        1. Run insights-client with an empty --output-file option
+        2. Verify that the command fails with a return code of 1
+        3. Verify that the error message indicates that --output-file cannot be empty
+    :expectedresults:
+        1. The insights-client runs with an empty option
+        2. The command fails with a return code of 1
+        3. The error message is as expected
+    """
     cmd_result = insights_client.run("--output-file=", check=False)
     assert cmd_result.returncode == 1
     assert "ERROR: --output-file cannot be empty" in cmd_result.stderr
 
 
 def test_output_specifying_both_dir_and_file(insights_client, tmp_path):
-    """Test both --output-file and --output-dir (with valid parameters) at once.
-    This should fail with error message suggesting to specify only one at a time.
+    """
+    :id: a572654d-904c-410e-ba00-f7b63f910e53
+    :title: Test Output Specifying Both Directory and File
+    :description:
+        Verifies that specifying both output file and output directory
+        option together fails with an appropriate error message
+    :tier: Tier 1
+    :steps:
+        1. Define both output file and directory paths
+        2. Run insights-client with --output-file and --output-dir
+        3. Verify the command fails with an error code of 1
+        4. Verify the error message is appropriate and as expected
```

Comment:
Should this step be more specific? `Verify the error message says they can't be used together` or something?